src/tools/sdk/dart_sdk_archive/lib/src/run_app.dart

```diff
@@ -32,7 +31,10 @@ Future<void> runApp() async {
     querySelector('#dev-os') as SelectElement,
   );
 
-  unawaited(stableSelector.init());
-  unawaited(betaSelector.init());
-  unawaited(devSelector.init());
+  // ignore: unawaited_futures
```

Comment:
I have a vague recollection that `unawaited` was on it's way into the core libraries? @johnpryan probably knows this better than I do.

Reply:
Yep, but it won't be added until at least the next stable release, so don't want to rely on it yet. Here's a link to the new `unawaited` docs: https://api.dart.dev/dev/dart-async/unawaited.html